Sources/Controllers/2.0/OAMapSettingsMainScreen.mm

```diff
@@ -554,6 +554,8 @@ - (void) overlayChanged:(id)sender
     UISwitch *switchView = (UISwitch *)sender;
     if (switchView)
     {
+        _settings.isOverlayLayerVisible = switchView.isOn;
```

Comment:
It is not clear why do we need additional settings isOverlayLayerVisible and isUnderlayLayerVisible

Reply:
Removed these variables and all the methods that change them.  Now _app.data.lastOverlayMapSource is used instead.